source/val/validation_state.h

```diff
@@ -653,6 +653,19 @@ class ValidationState_t {
   spv_result_t CooperativeMatrixShapesMatch(const Instruction* inst,
                                             uint32_t m1, uint32_t m2);
 
+  // Returns true if the edge |source| -> |dest| is a structured exit of the
```

Comment:
This looks mysteriously different from the other descriptions of structured exits.

Reply:
It also has no definition. Thanks for noticing. It was from an early implementation before it evolved into what it is now. I'm removing it.

Reply:
Yay for code review!